### ISSUE TITLE
Release 4.25.16p1 as 4.25.16.1 to RubyGems?

### ISSUE THREAD

**Issue by dmke** [sisimai/rb-sisimai#257]
Yesterday, after the release of 4.25.16p1, I wanted to update [sisimai-web](https://github.com/digineo/sisimai-web)  accordingly, but was surprised that [RubyGems doesn't list that version](https://rubygems.org/gems/sisimai/versions):

![image](https://github.com/sisimai/rb-sisimai/assets/327411/d8f2c3ff-38c0-441d-9ab3-bf81321c70bf)

In general, this is not much of a problem, because I can just change the Gemfile entry (cf. [a934ad0a](https://github.com/digineo/sisimai-web/commit/a934ad0a0e4ceeaa2cd46386073b15379c58db88)):

```ruby
# old
gem "sisimai"

# new
git_source(:github) { |name| "https://github.com/#{name}.git" }
gem "sisimai", github: "sisimai/rb-sisimai", ref: "v4.25.16p1"
```

However, today I got [a PR from dependabot](https://github.com/digineo/sisimai-web/pull/40), which seems to interpret the `p1` suffix as some kind of pre-release (Semver defines MAJOR "." MINOR "." PATCH ["-" prerelease] ["+" build]), and dutifully "upgrades" to 4.25.16...

So, long story short: Would you mind releasing 4.25.16p1 as 4.25.16.1 to RubyGems?

**Comment by dmke** [comment]
It looks like v5.0.0 didn't make it to RubyGems as well. :thinking:

**Comment by azumakuniyuki** [comment]
@dmke Thanks for the report.
# v4.25.16p1
Now, I am currently busy with "No Auth, No Entry" related tasks, so I will release `v4.25.16p1` (or `p2` ?) to RubyGems next week. 

If you need v4.25.16p1 before it is released to RubyGems, you can get it via `git clone -b 4-stable https://github.com/sisimai/rb-sisimai.git`.

# v5.0.0
> It looks like v5.0.0 didn't make it to RubyGems as well. 🤔

We are currently working on the [README.md](https://github.com/sisimai/rb-sisimai/blob/5-stable/README.md) and documentation in [libsisimai.org](https:/libsisimai.org/) for Sisimai 5, and we plan to release `v5.0.0` to RubyGems once they are completed. This is expected to take a few weeks.

Best regards,

**Comment by azumakuniyuki** [comment]
The README for the Perl version of Sisimai 5 is available at https://github.com/sisimai/p5-sisimai/pull/500/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5, although it has not been merged yet. The functionality is compatible with the Ruby version, so please refer to it if you find it helpful. :-)
